app/(info-page)/inclusion/page.tsx

```diff
@@ -1,5 +1,12 @@
+import TitleSectionInclusion from '@/app/(info-page)/_components/TitleSectionInclusion'
+
 const Inclusion = () => {
-  return <div>inclusion</div>
+  return (
+    <div className='min-h-screen relative px-20 py-28'>
```

Comment:
there has been an update with a general main layout (in layout.tsx) that will clash with the inline padding you have here. i would suggest to merge in main to get the new updates. check out the page.tsx in faq where ida did a similar section if you want to see her way of adapting to the general layout

Reply:
I've done the changes, I will wait for main to be fixed and then pull it down to my local, and check all changes.